src/components/Routing.js

```diff
@@ -173,14 +191,19 @@ class Routing extends Component {
                                     path='/create/:fundingContract?/:fundingKey?'
                                     component={CreateAccountWithRouter}
                                 />
-                                <PrivateRoute
+                                <Route
                                     exact
-                                    path='/set-recovery/:accountId'
+                                    path='/set-recovery/:accountId/:isNew?/:fundingContract?/:fundingKey?'
```

Comment:
@mattlockyer after reviewing both contract helper and wallet I am not sure why we have to pass `isNew` everywhere. Isn't this info basically on chain already?

It usually helps to not have additional source of truth for the same thing, as then easier to get inconsistent state.

Reply:
Sure we can deprecate this in favor of checking if the account exists on chain, that's a good idea.

I will have to follow up with a fix after release.

Reply:
Will have to be a PR fix after this release.